roles/container_runtime/defaults/main.yml

```diff
@@ -23,7 +23,7 @@ openshift_docker_ent_reg: 'registry.access.redhat.com'
 
 openshift_docker_options: False  # str
 openshift_docker_log_driver: False  # str
-openshift_docker_log_options: []
+openshift_docker_log_options: ["max-size=50m"]
```

Comment:
Looks like this causes drivers other than json (and possibly some others?) to fail:

 "line": "OPTIONS=' --selinux-enabled    --log-opt max-size=50m    --log-driver=journald --signature-verification=False'", 

Need to come up with a way to account for this.

Also, prefer yaml style list instead of json
```yaml
some_list:
- item1
```

Reply:
Changed to yaml style list. I am sorry, but why/how would it cause some drivers to fail?

Reply:
IMHO, when configuring other drivers, it's the admin's responsibility to choose the correct settings.  For example, in the ``journald`` case, the log rotation/size settings are totally outside the control of ``openshift``.

The installer would be forever "chasing its tail", if it must synchronize with docker's log option processing logic, for each driver.